metadata-ingestion/src/datahub/ingestion/source/sql/mariadb.py

```diff
@@ -7,14 +15,112 @@
     support_status,
 )
 from datahub.ingestion.source.sql.mysql import MySQLConfig, MySQLSource
+from datahub.ingestion.source.sql.stored_procedures.base import (
+    BaseProcedure,
+)
+
+# MariaDB inherits shared stored procedure config from MySQL via StoredProcedureConfigMixin
+
+logger: logging.Logger = logging.getLogger(__name__)
+
+STORED_PROCEDURES_QUERY = """
```

Comment:
Both MySQL and MariaDB define STORED_PROCEDURES_QUERY with slightly different column selections:
  - MySQL (mysql.py:80-93): Selects lowercase aliases (name, definition, comment)
  - MariaDB (mariadb.py:26-40): Selects uppercase aliases (ROUTINE_SCHEMA, ROUTINE_NAME, etc.)
  
  Is this correct?